code/modules/mob/hear_say.dm

```diff
@@ -68,8 +70,9 @@
 			else if(!is_blind())
 				to_chat(src, "<span class='name'>[speaker_name]</span>[alt_name] talks but you cannot hear \him.")
 	else
+		var/message_to_send = null
+		var/nverb = verb
```

Comment:
There's no reasons to move it

Reply:
idk whats the original position and theres no reason to move it back :trollface:

Reply:
You literally see old positions on the github. It was right under `if(language)`

Reply:
ok i moved em